### ISSUE TITLE
Icon Request: Video game Console (xbox, playstation/psn, nintendo)

### ISSUE THREAD

**Issue by begedin** [FortAwesome/Font-Awesome#1006]
I started using Font Awesome on my blog and it's, well, awesome, but I'd love to have a set of game console/platform icons I could use. I'm talking about stuff like PC, PlayStation, DS, etc., possibly even distribution platforms, such as Steam, GOG, etc.

I tried using the search for this, but I got nothing. Sorry if I'm just repeating another issue.

**Comment by IngwiePhoenix** [comment]
+1. Working on a site with user profiles that have PSN and XBL in them. Would be great to have this.

**Comment by bramv123** [comment]
+1

**Comment by cvsguimaraes** [comment]
+1

**Comment by Arisae** [comment]
+1 i want it!

**Comment by ahmadmmughal** [comment]
+1! really need these!

**Comment by nilbenis** [comment]
+1

**Comment by elavender** [comment]
+1

**Comment by justinlosh** [comment]
+1

**Comment by omstuff** [comment]
+1

**Comment by Techokami** [comment]
+1 for the Xbox and Nintendo Network icons. Currently I'm using the Windows logo on my tumblr for Xbox Live, and I would like to add in a link to my Nintendo Network/Miiverse profile.

**Comment by G-Driver** [comment]
+1

**Comment by BobisOnlyBob** [comment]
+1

**Comment by bumbleblym** [comment]
:+1:

**Comment by geeklore** [comment]
+1

**Comment by Ludophiles** [comment]
+1

**Comment by bladefist** [comment]
+1

**Comment by markusdrake** [comment]
+1

**Comment by ishotjr** [comment]
:+1:

**Comment by orugari** [comment]
+1

**Comment by psychoph** [comment]
+1 would love this.

**Comment by DennaGherlyn** [comment]
+1 
Especially for the distribution platforms. Steam is already there but I'm missing GOG, Desura and Origin.

**Comment by AustinDizzy** [comment]
+1

**Comment by pbhogan** [comment]
+1

**Comment by QFGlenn** [comment]
TheChen5 commented 18 hours ago

First of all, I think this violates Font Awesome rules to not request a set of icons at once.

**Comment by DennaGherlyn** [comment]
"Please make requests for single icons, unless you are requesting a couple of strictly related icons"
Are the requested icons not related?

**Comment by tagliala** [comment]
This is an old request and has a lot of +1s.

Major Browser were all included in a single request, see #219 

IMHO it's fine to have the 3 major video game console brands in a single request.

**Comment by syntactic-salt** [comment]
+1

**Comment by Zaanmedia** [comment]
+1

**Comment by d1p112** [comment]
+1

**Comment by nDiVinEx** [comment]
+1

**Comment by ncpierson** [comment]
+1

**Comment by MadTiago** [comment]
+1

**Comment by Crob0** [comment]
+1

**Comment by peeweejd** [comment]
+1

**Comment by FBI23** [comment]
+1

**Comment by nilthacker** [comment]
+1

**Comment by codyn329** [comment]
+1

**Comment by Turbotailz** [comment]
+1, for Xbox especially. Really wanting to display a list of networks people can connect with me on and Xbox is a huge one for me.

You have included Steam, why not Xbox or PS?

**Comment by gesteves91** [comment]
+1

**Comment by drvieira** [comment]
+1

**Comment by michaeldtaylor** [comment]
+1

**Comment by lenppqq** [comment]
+1 for Xbox!

**Comment by Sobient** [comment]
+1
NES, SNES, SEGA Megadrive, PCX, C64 and N64 for starters.

**Comment by robatato** [comment]
YAS PLZ +1

**Comment by chief93** [comment]
+1 fa-xbox plz =)

**Comment by slyscorpion** [comment]
+1 fa-gogcom please!
![goggalaxy 1 0 2 958](https://cloud.githubusercontent.com/assets/12873013/13838782/fd50ba1c-ec16-11e5-86a5-0d15266656f6.png)

**Comment by webbson** [comment]
Wow that these hasn't been added 3 years later. 
Also +1.

**Comment by gviscardi** [comment]
+1

**Comment by tatoufo** [comment]
+1

**Comment by tryton-vanmeer** [comment]
+1

**Comment by gislikonrad** [comment]
+1

**Comment by g33kidd** [comment]
+5

**Comment by psychoph** [comment]
Did we level this up yet? Am I hearing legend of zelda music?

**Comment by kincaider** [comment]
+1

**Comment by daygon2007** [comment]
+1

**Comment by pfeiferchristopher** [comment]
+1

**Comment by Kovah** [comment]
I wonder why these icons are not already implemented? I think they are pretty important.
Any way to help here?

**Comment by jamesfmackenzie** [comment]
+1

**Comment by lmeyer** [comment]
+1

**Comment by nfMalde** [comment]
Hello,

I've read an interesting report lately which is based on hard facts, you may like it I guess, read it here <http://xerdoltucra.niblet.us/e2hubkwo>

See you around, malte@networkfreaks.de

**Comment by g33kidd** [comment]
This issue is so popular it's began to attract spam +1s...

**Comment by Sobient** [comment]
@tagliala It's about time to do something about this issue. It's very clear that alot of people want these icons included. In fact it's becoming so popular  it's attracting spam bots now, spamming those who have commented. I suggest you either promise to include this in version X or close the issue down entirely (if you do not plan to include these in FontAwesome). 

Either way, the time to do something is -now-. Make it happen.

**Comment by nfMalde** [comment]
Hi,


I've accidentaly found that crazy stuff, it is just  so cool! You've got to see that! Check  it out here <http://next.hrcflorida.com/e2ygxu>


See you around, malte@networkfreaks.de

**Comment by nfMalde** [comment]
Hi,

Here is  some information on the new stuff that may seem  helpful to you, read more here <http://name.riverwing.net/e2rjvstp>

In haste, malte@networkfreaks.de

**Comment by Kovah** [comment]
@nfMalde Stop posting your spam! 😠 

@davegandy Please lock comments on this thread.

**Comment by n8-sd** [comment]
@Kovah 

Let me guess this is a licensing issue perhaps?

Xbox (Microsoft) and Playstation (Sony) not give consent?

Pretty poor solution ( Locking comments) instead of talking to people about why something hasn't / can't be done.

**Comment by Kovah** [comment]
@n8-sd Whoops. I think you got something wrong. Github just deleted three spam comments which my comment was about. I updated my post.

**Comment by n8-sd** [comment]
@Kovah thats a better reasoning.

How ever for something such a hot topic, I dont think it should be "locked"

**Comment by JB-MIAGE** [comment]
+1

**Comment by willoughby** [comment]
+1

**Comment by Maelstromeous** [comment]
+1

**Comment by dartrix** [comment]
+1

**Comment by niarora** [comment]
+ fa-xbox / ps

**Comment by kel** [comment]
This would be a great addition for FA5!

**Comment by LoliN3k0** [comment]
And now we have a new videoconsole icon. Nintendo Switch

https://cdn02.nintendo-europe.com/media/images/11_square_images/systems_12/nintendo_switch/SQ_NintendoSwitchLogo_sharing_image_400.png

I hope its in in FA5... More than 3 years about this request and nothing :(

**Comment by vsential** [comment]
+1

**Comment by GazCBG** [comment]
+1 for xbox icon

**Comment by soulis3** [comment]
+1

**Comment by GNSubrahmanyam** [comment]
+1

**Comment by GNSubrahmanyam** [comment]
+1

**Comment by Re-1** [comment]
+1

**Comment by Galkon** [comment]
+1

**Comment by visualpharm** [comment]
Guys, it's painful to watch it, bunch of people asking for a single icon for year. It's 30 min of work after all.

Let's say they're licensed under [Good Boy License](https://icons8.com/good-boy-license/). If you're enterprise and your lawyers hate broad terms, let's double-license it under MIT.

There are [Xbox](https://icons8.com/icon/12473/xbox) and [PlayStation](https://icons8.com/icon/12461/playstation) icons.

I'm beginning a campaign, giving away the top requests for FA icons for free, open source. You can incorporate them in a number of ways: generating another icon font, svg set, or (my preference) [compiling](https://iconpharm.com) all FA and third party icons into a single file.

Previous ones were [moderator](https://github.com/FortAwesome/Font-Awesome/issues/2553#issuecomment-289439382),  [discord](https://github.com/FortAwesome/Font-Awesome/issues/7574#issuecomment-289875669), [coins](https://github.com/FortAwesome/Font-Awesome/issues/1400#issuecomment-292190852), [Google Play](https://github.com/FortAwesome/Font-Awesome/issues/2272#issuecomment-290106280),  [thin Close](https://github.com/FortAwesome/Font-Awesome/issues/1540#issuecomment-302573662), [walking man](https://github.com/FortAwesome/Font-Awesome/issues/844#issuecomment-302574446), [t-shirt](https://github.com/FortAwesome/Font-Awesome/issues/715), [Facebook messenger](https://github.com/FortAwesome/Font-Awesome/issues/6912#issuecomment-302864830) and [Line](https://github.com/FortAwesome/Font-Awesome/issues/2631#issuecomment-292290534) icons.

Please let me know if you want me keep going these icon requests, or compile some upgrade file, or how should we do it?

**Comment by ewersp** [comment]
++1

**Comment by markadrake** [comment]
@visualpharm deserves a big thumbs up. This is a painful thread to have notifications turned on to. Been watching it for years. And it'll be years still until Font Awesome adds them (if they ever do) and only after we've all moved to SVG sprites (if you haven't, you should).

**Comment by regimbal93** [comment]
+1

**Comment by tagliala** [comment]
Hi, thanks for all your reactions to this request!

Playstation, Xbox and Nintendo switch will be included in the upcoming Font Awesome 5 Free. 🎉

Note that Nintendo logo does not appear to have an usable icon, so it will not be included.

I'm going to close this request for management purposes

Regards

**Comment by paulocrash** [comment]
+1

**Comment by SensehacK** [comment]
+1
